pkg/runtime/runtime.go

```diff
@@ -628,6 +628,8 @@ func (a *DaprRuntime) getSubscribedTopicsFromApp() []string {
 			topics = resp.Topics
 		}
 	}
+
+	log.Printf("App is subscribed to the following topics: %v", topics)
```

Comment:
Do you really need this change or is it for debugging? If this change is not needed, just revert this file.

Reply:
I think this is something the runtime *should* trace, but just didn't before.  And yes it will help debuggability later and in production.